### PR TITLE
 Update references to the closed bungeecord PRs for SpongeForge support 

### LINKED ISSUE #748: Mentioned BungeeCord Issues have been closed
[Docs: `bungeecord.html`](https://docs.spongepowered.org/stable/en/server/getting-started/bungeecord.html?highlight=Pull%20requests#ip-forwarding) | [Source: `bungeecord.rst`](https://github.com/SpongePowered/SpongeDocs/blame/stable/source/server/getting-started/bungeecord.rst#L39)

> For SpongeVanilla or Vanilla clients connecting to unmodded SpongeForge, IP Forwarding works out of the box. [Pull](https://github.com/SpigotMC/BungeeCord/pull/1557) [requests](https://github.com/SpigotMC/BungeeCord/pull/1678) have been supplied to BungeeCord to allow BungeeCord to support SpongeForge natively. We are awaiting it to be included in the main product. For now please use one of the solutions mentioned above.

The linked issues have been closed, so we cannot keep the as "awaited". We should probably either update the references to their successor or rephrase it like: `These contributions have been rejected, so you don't have any other choice than use one of the solutions mentioned above.`

Asking @dualspiral for insider knowledge.

### PR DESCRIPTION
**Pull request by ST-DDT**
Fixes #748

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/e95efaf/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/e95efaf/server/getting-started/bungeecord.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/751#issuecomment-415873188), not to the direct url to the preview._

File: source/server/getting-started/bungeecord.rst [hunk 1]
```diff
@@ -36,10 +36,9 @@ connected server, even though the servers are being run in offline mode.
  products, although they are developed by trusted members of the community.
 
 For SpongeVanilla or Vanilla clients connecting to unmodded SpongeForge, IP Forwarding works out of the box.
-`Pull <https://github.com/SpigotMC/BungeeCord/pull/1557>`_
-`requests <https://github.com/SpigotMC/BungeeCord/pull/1678>`_ have been supplied to BungeeCord to allow BungeeCord to
-support SpongeForge natively. We are awaiting it to be included in the main product. For now please use one of the
-solutions mentioned above.
+Sponge supplied `fi <https://github.com/SpigotMC/BungeeCord/pull/1557>`_\
+`xes <https://github.com/SpigotMC/BungeeCord/pull/1678>`_ to BungeeCord for modded SpongeForge support, but these
+haven't been merged since. So use one of the solutions mentioned above.
```
Comment (phit) [inline source/server/getting-started/bungeecord.rst]: > haven't been merged. Please use one of the solutions mentioned above.

**Review by phit — CHANGES_REQUESTED** [review]

**Review by phit — APPROVED** [review]

File: source/server/getting-started/bungeecord.rst [hunk 2]
```diff
@@ -36,10 +36,9 @@ connected server, even though the servers are being run in offline mode.
  products, although they are developed by trusted members of the community.
 
 For SpongeVanilla or Vanilla clients connecting to unmodded SpongeForge, IP Forwarding works out of the box.
-`Pull <https://github.com/SpigotMC/BungeeCord/pull/1557>`_
-`requests <https://github.com/SpigotMC/BungeeCord/pull/1678>`_ have been supplied to BungeeCord to allow BungeeCord to
-support SpongeForge natively. We are awaiting it to be included in the main product. For now please use one of the
-solutions mentioned above.
+Sponge supplied `fi <https://github.com/SpigotMC/BungeeCord/pull/1557>`_\
```
Comment (dualspiral) [inline source/server/getting-started/bungeecord.rst]: The fixes were independent of the Sponge team. I did them of my own accord way before I was a member of Sponge, so it is inaccurate to say as such. I'd go with:

"`Pull <https://github.com/SpigotMC/BungeeCord/pull/1557>` `requests <https://github.com/SpigotMC/BungeeCord/pull/1678>` were made to BungeeCord for modded SpongeForge support, but these haven't been accepted. Other solutions (mentioned above) include these patches and should be used instead."

Comment (Inscrutable) [reply source/server/getting-started/bungeecord.rst]: As long as the link ReST formatting is correct.

Comment (dualspiral) [reply source/server/getting-started/bungeecord.rst]: Yep, sorry, was writing this from bed last night on my tablet!

**Review by dualspiral — APPROVED** [review]

**Pull request merged**